examples/teleop_ros2/python/teleop_ros2_publisher.py

```
@@ -549,145 +549,129 @@ def __init__(self) -> None:
-                            if wrist_tfs:
-                                self._tf_broadcaster.sendTransform(wrist_tfs)
-                        elif self._mode == "controller_teleop":
-                            ee_msg = _build_ee_msg_from_controllers(
-                                left_ctrl, right_ctrl, now, self._world_frame
+                            finger_joints_msg.position = np.concatenate(
+                                [left_arr, right_arr]
+                            ).tolist()
+                            self._pub_finger_joints.publish(finger_joints_msg)
+
+                    if self._mode in ("controller_raw", "full_body"):
+                        if not left_ctrl.is_none or not right_ctrl.is_none:
```

Ruff (0.15.6)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)